spec/server_spec.cr

```diff
@@ -141,15 +141,6 @@ describe AvalancheMQ::Server do
     end
   end
 
-  it "should mark the vhost as dirty when a queue is purged" do
```

Comment:
Can we write another test for this? but one that checks that bytes are actually recovered from disk or something?

Reply:
Yes, some type of test that verifies that GC has run. Iäll take a look at it.